openstates/fl/bills.py

```diff
@@ -461,7 +463,11 @@ def handle_page(self):
         # set voters
         for vtype, voters in votes.items():
             for voter in voters:
-                vote.vote(vtype, voter)
+                voter = voter.strip()
+                if "  VA" in voter:
```

Comment:
this seems like it could cause a weird error later, can we just replace the "VA" or at least comment what is going on here?

Reply:
Added in a comment explaining why I remove the VA and the 2 extra spaces. Here is an example name that this if statement takes care of:
"Cruz                                                               VA"